src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceConnection.java

```diff
@@ -0,0 +1,19 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.WebDriver;
+
+/**
+ * SalesForceConnection.java
+ * Class to initialize WebDriver for all pages.
+ */
+public class SalesForceConnection {
+    protected WebDriver webDriver;
+
+    /**
+     * Constructor.
+     */
+    public SalesForceConnection() {
+        this.webDriver = DriverManager.getInstance().getNavigator();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)